src/HeckeTypes.jl

```diff
@@ -371,6 +403,11 @@ mutable struct SMatSpace{T} <: Ring
   end
 end
 
+"""
+  SMat{T}
```

Comment:
```suggestion
    SMat{T}
```

Reply:
when the test have passed I accept that change and merge